python/ray/serve/api.py

```diff
@@ -500,7 +413,6 @@ def list_deployments() -> Dict[str, Deployment]:
 def run(
     target: Union[ClassNode, FunctionNode],
     _blocking: bool = True,
-    *,
     host: str = DEFAULT_HTTP_HOST,
     port: int = DEFAULT_HTTP_PORT,
```

Comment:
we need the place to specify other options here in `serve.run` but that can be added as follow up API

Reply:
(without breaking changes)